modules/core/src/main/java/org/locationtech/jts/index/intervalrtree/SortedPackedIntervalRTree.java

```diff
@@ -132,12 +127,7 @@ private void buildLevel(List src, List dest)
 			}
 		}
 	}
-	
-  private void printNode(IntervalRTreeNode node)
-  {
-    System.out.println(WKTWriter.toLineString(new Coordinate(node.min, level), new Coordinate(node.max, level)));
-  }
-  
```

Comment:
why is this removed?

Reply:
Because this is a dead code, method never invoked.

Reply:
Just comment it out then.  It may be used for debugging in the future.

Reply:
fixed.